package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "dependencies": {
     "@apollo/react-hooks": "^3.1.3",
-    "@gnosis.pm/safe-apps-sdk": "0.4.1",
+    "@gnosis.pm/safe-apps-sdk": "https://github.com/gnosis/safe-apps-sdk.git#ea8dcab",
```

Comment:
v0.4.2 is being released right now with the necessary changes. I would use that version.

Reply:
it's not yet on npm

Reply:
@dasanra done!